<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindowClass</class>
 <widget class="QDialog" name="SearchWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchWindow</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>16</y>
     <width>651</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Path: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="7">
     <widget class="QLineEdit" name="pathLine"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Type: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="3">
     <widget class="QLineEdit" name="typeLine"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Time created:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateTimeEdit" name="minCreatedTime"/>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="timeCreatedMinCheckBox">
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QDateTimeEdit" name="maxCreatedTime"/>
    </item>
    <item row="2" column="7">
     <widget class="QCheckBox" name="timeCreatedMaxCheckBox">
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Time edited:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDateTimeEdit" name="minEditedTime"/>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="timeEditedMinCheckBox">
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QDateTimeEdit" name="maxEditedTime"/>
    </item>
    <item row="3" column="7">
     <widget class="QCheckBox" name="timeEditedMaxCheckBox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Hash: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="7">
     <widget class="QLineEdit" name="hashLine"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Custom metadata:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="5">
     <widget class="QLineEdit" name="metadataLine"/>
    </item>
    <item row="5" column="6" colspan="2">
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="8">
     <widget class="QTableWidget" name="resultsTable"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
